### ISSUE TITLE
Add support for different output color spaces (at least sRGB) in addition to CMYK

### ISSUE THREAD

**Issue by Nivekvb** [sketch-hq/print-export-sketchplugin#6]
I've got a design file with default color profile 'unmanaged'
Where #949bff turns into  #8097C1.

After changing it to 'sRGB'
Where #949bff also turns into  #8097C1.

I also have  a design file with default color profile 'sRGB' fromt the start.
Where #949bff also turns into  #8097C1.

The normal export artboard to PDF works fine on both files. Why should this function differently?
So I can't really use the print option to share designs with a client...

PS: love your work!

**Comment by sylrs** [comment]
Because you're exporting your Sketch file to print, the colors change from RGB to CMYK. Due to the nature of combining those 4 colors (cyan, magenta, yellow, and black) in ink, many bright and saturated colors are difficult or impossible to recreate.

**Comment by olios** [comment]
> Due to the nature of combining those 4 colors (cyan, magenta, yellow, and black) in ink, many bright and saturated colors are difficult or impossible to recreate.
Request or solution:

> 

- is there a way to control this colour transformation so we can adjust to a cmyk colour closest from Pantone or other print standard for plain inks as well. 
- Pantone is one standard inks provider to print but others exist, different like in Asia or other regions.

**Comment by Tafkadasoh** [comment]
I was looking for an sRGB export option, too. As a temporary fix for myself, I've forked the repository and changed the code in such way, that the export is created within the sRGB color space.

See https://github.com/Tafkadasoh/print-export-sketchplugin

**Comment by bomberstudios** [comment]
Indeed, the plugin is converting colors to CMYK (it's a _print_ export plugin, after all) so some color shifting is unavoidable.

We could consider adding an option to choose the target color space, so I'll tag this as a request and update the title.

**Comment by Tafkadasoh** [comment]
An option to switch between both color spaces would be really helpful! I wanted to update my sRGB fork (cf. link above) with such an option, but I don't find the time for it in the near future.
